pkg/util/rest/mapper.go

```diff
@@ -1,4 +1,4 @@
-package client
+package rest
```

Comment:
How about mimicking what upstream does and have this in `pkg/api/meta`? @deads2k

Reply:
@soltysh i don't care that much about location of this... i'm fine moving that into pkg/api/meta just felt like this is related to "rest/clients" instead of api

Reply:
> How about mimicking what upstream does and have this in pkg/api/meta? @deads2k

This function should die.  We don't want to maintain our own restmappers if we can avoid it.